modules/pod-disruption-eviction-policy.adoc

```diff
@@ -15,15 +15,10 @@ You can choose one of the following policies:
 IfHealthyBudget:: Running pods that are not yet healthy can be evicted only if the guarded application is not disrupted.
 
 AlwaysAllow:: Running pods that are not yet healthy can be evicted regardless of whether the criteria in the pod disruption budget is met. This policy can help evict malfunctioning applications, such as ones with pods stuck in the `CrashLoopBackOff` state or failing to report the `Ready` status.
-
-:FeatureName: Specifying the unhealthy pod eviction policy for pod disruption budgets
-include::snippets/technology-preview.adoc[]
-
-To use this Technology Preview feature, you must have enabled the `TechPreviewNoUpgrade` feature set.
-
-[WARNING]
++
+[NOTE]
 ====
-Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
+It is recommended to set `AlwaysAllow` to your PodDisruptionBudgets to support eviction of misbehaving applications during a node drain.
```

Comment:
IMO I would add even the second part: `The default behavior is to wait for the application pods to become healthy before the drain can proceed.` or hint it can potentially have negative implications for the application/

Reply:
Added, thank you!